be/src/runtime/tablets_channel.cpp

```diff
@@ -104,20 +102,20 @@ Status TabletsChannel::add_batch(const PTabletWriterAddBatchRequest& params) {
     return Status::OK();
 }
 
-Status TabletsChannel::close(int sender_id, bool* finished,
+Status TabletsChannel::close(int sender_idx, bool* finished,
```

Comment:
why change sender_id to sender_idx? I think if there is no strong reason, better to keep the original name.